app/src/pages/OnDeviceDisplay/ProtocolSetup/__tests__/ProtocolSetup.test.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react'
 import { Route } from 'react-router'
 import { MemoryRouter } from 'react-router-dom'
 import { when, resetAllWhenMocks } from 'jest-when'
+import { UseQueryResult } from 'react-query'
```

Comment:
this can be before `import type { DeckConfiguration,`
```suggestion
import type { UseQueryResult } from 'react-query'
```